### PR TITLE
D08943012 PA2 Submission

### PR DESCRIPTION
**Pull request by William324-Hsieh**

### CODE REVIEW

**Comment by nianzelee** [comment]
@William324-Hsieh Please do not check in dotfiles (.nfs) and dependency files (.d). Please add the reference answers back.

**Review by nianzelee — CHANGES_REQUESTED** [review]
Add the reference answers of pa1 back, remove .csv file and pa1-ref binary file.

**Comment by William324-Hsieh** [comment]
PA1 ref在Merge branch不是就倍加會去了嗎?

File: Makefile [hunk 1]
```diff
@@ -56,7 +56,8 @@ ARCHFLAGS := $(ARCHFLAGS)
 
 OPTFLAGS  ?= -g -O
 
-CFLAGS    += -Wall -Wno-unused-function -Wno-write-strings -Wno-sign-compare $(ARCHFLAGS)
+CFLAGS    += -O3 $(ARCHFLAGS)
```
Comment (nianzelee) [inline Makefile:59]: Typically a feature branch (your branch) should not change the compilation of a project, but I accept your change here.

**Pull request merged**